decidim-admin/app/permissions/decidim/admin/permissions.rb

```diff
@@ -83,6 +83,7 @@ def apply_global_moderations_permission_for_admin!
         return unless admin_terms_accepted?
         return unless permission_action.subject == :global_moderation
         return allow! if user.admin?
+        return disallow! if user_valuator?
```

Comment:
Maybe I'm a bit rusty after my holidays 😅  but I think this actually isn't being used. If I remove this line, the specs still pass (ie `bin/rspec decidim-debates/spec/permissions/`)

By using `git blame`  I see that was introduced on a merge with `develop` (f7b31487a311f078590ecd8a0ea9079381a06571). Also according to the method name (`apply_global_moderations_permission_for_admin`), it seems like it isn't related to this PR but to #10204.

Is this actually being used? If so can you provide an spec? Thanks! 

(As I mention, full disclosure that I may be a bit rusty, so sorry in advance if that's the case 🙈)

Reply:
That line is actually being used. if we remove that, we will start having the permission specs in decidim-assemblies, decidim-conferences decidim-participatory_processes failing ... 

An example: 
```
rspec ./spec/permissions/decidim/assemblies/permissions_spec.rb[1:1:2:1:3:1:1] # Decidim::Assemblies::Permissions when the action is for the public part when accessing global moderation behaves like access for roles when user is a space collaborator behaves like access for role 
rspec ./spec/permissions/decidim/assemblies/permissions_spec.rb[1:1:2:1:1:1:1] # Decidim::Assemblies::Permissions when the action is for the public part when accessing global moderation behaves like access for roles when user is org admin behaves like access for role 
rspec ./spec/permissions/decidim/assemblies/permissions_spec.rb[1:1:2:1:2:1:1] # Decidim::Assemblies::Permissions when the action is for the public part when accessing global moderation behaves like access for roles when user is a space admin behaves like access for role 
rspec ./spec/permissions/decidim/assemblies/permissions_spec.rb[1:1:2:1:5:1:1] # Decidim::Assemblies::Permissions when the action is for the public part when accessing global moderation behaves like access for roles when user is a space valuator behaves like access for role 
rspec ./spec/permissions/decidim/assemblies/permissions_spec.rb[1:1:2:1:4:1:1] # Decidim::Assemblies::Permissions when the action is for the public part when accessing global moderation behaves like access for roles when user is a space moderator behaves like access for role 
```

Reply:
I can't reproduce that error. Steps to check it out: 

1. Checkout the branch: 

```bash
$ gh pr checkout 10205 
```

2. Remove the method call and the method: 

```diff
diff --git a/decidim-admin/app/permissions/decidim/admin/permissions.rb b/decidim-admin/app/permissions/decidim/admin/permissions.rb
index 30dfe028ef..feaa562e55 100644
--- a/decidim-admin/app/permissions/decidim/admin/permissions.rb
+++ b/decidim-admin/app/permissions/decidim/admin/permissions.rb
@@ -83,7 +83,6 @@ module Decidim
         return unless admin_terms_accepted?
         return unless permission_action.subject == :global_moderation
         return allow! if user.admin?
-        return disallow! if user_valuator?
 
         return allow! if Decidim.participatory_space_manifests.flat_map.any? do |manifest|
           Decidim
@@ -97,20 +96,6 @@ module Decidim
         disallow!
       end
 
-      def user_valuator?
-        return unless user
-        return unless user.admin?
-
-        Decidim.participatory_space_manifests.flat_map.any? do |manifest|
-          Decidim
-            .find_participatory_space_manifest(manifest.name)
-            .participatory_spaces
-            .call(user.organization)&.any? do |space|
-            space.respond_to?(:user_roles) && space.user_roles(:valuator).where(user:).any?
-          end
-        end
-      end
-
       def apply_newsletter_permissions_for_admin!
         return unless admin_terms_accepted?
         return unless permission_action.subject == :newsletter
```

3. Run all the permissions specs:

```bash
$ for file in $(find decidim*/spec/permissions/ -name "*.rb") ; do echo $file; bin/rspec $file || break; done  
``` 

All the specs are passing.

In the case of `decidim-assemblies`:

```bash
$ bin/rspec decidim-assemblies/spec/permissions/decidim/assemblies/permissions_spec.rb 

(...)

Finished in 19.23 seconds (files took 3.51 seconds to load)
128 examples, 0 failures

Randomized with seed 31897

Coverage report generated for RSpec to /home/apereira/Work/decidim/decidim/coverage. 7486 / 98357 LOC (7.61%) covered.
```